lib/components/button.vue

```diff
@@ -18,7 +20,7 @@
                 ];
             },
             componentType() {
-                return (this.href || this.to) ? 'b-link' : 'button';
+                return (this.href || this.to) ? bLink : 'button';
```

Comment:
@gpyh, 
comparing to #174, this peace specially omitted or forgotten?

        components:{
            bLink
        },